I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 37, in script
    $ p = renpy.input("Oh, um I am. . .", color="#fff")
  File "game/script.rpy", line 37, in <module>
    $ p = renpy.input("Oh, um I am. . .", color="#fff")
TypeError: input() got an unexpected keyword argument 'color'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 37, in script
    $ p = renpy.input("Oh, um I am. . .", color="#fff")
  File "C:\Users\Amber0527\Documents\P. Programs\renpy-7.4.2-sdk/renpy/ast.py", line 922, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Amber0527\Documents\P. Programs\renpy-7.4.2-sdk/renpy/python.py", line 2218, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 37, in <module>
    $ p = renpy.input("Oh, um I am. . .", color="#fff")
TypeError: input() got an unexpected keyword argument 'color'

Windows-10-10.0.19041
Ren'Py 7.4.2.1292
Cranberry 1.0
Sun Nov 21 21:06:18 2021
